lms/djangoapps/cohort_management_dashboard/tab.py

```diff
@@ -11,5 +12,9 @@ class CohortManagementDashboardTab(EnrolledTab):
 
     @classmethod
     def is_enabled(cls, course, user=None):
-        return super(CohortManagementDashboardTab, cls).is_enabled(course, user)
+        if (
+            has_access(user, 'staff', course, course.id) or has_access(user, 'instructor', course, course.id)
+            or user.is_staff or user.is_superuser
+        ):
+            return super(CohortManagementDashboardTab, cls).is_enabled(course, user)
```

Comment:
Does None is expected value or method should return boolean?

Reply:
method should return boolean, this line returns True

Reply:
Ok, great! But if condition false method returns None.

Reply:
done